lib/index.js

```diff
@@ -460,20 +458,104 @@ function formatProperties(track, settings){
     || identify.userId()
     || identify.sessionId();
 
+  if (userAgent) extend(properties, formatUserAgent(userAgent));
+
+  // Map mobile specific special props
+  extend(properties, formatMobileSpecific(track));
+
+  // Map UTM params
+  if (campaign) {
+    properties.utm_name = campaign.name;
+    properties.utm_source = campaign.source;
+    properties.utm_medium = campaign.medium;
+    properties.utm_term = campaign.term;
+    properties.utm_content = campaign.content;
+  }
+
+  // Strip null/undefined values
   properties = reject(properties);
   stringifyValues(properties);
 
-  if (userAgent) {
-    var parsed = parse(userAgent);
-    var browser = parsed.browser
-    var os = parsed.os;
-    if (browser) properties.$browser = browser.name;
-    if (os) properties.$os = os.name;
-  }
   return properties;
 }
 
 /**
+ * Format userAgent properties
+ *
+ * https://mixpanel.com/help/questions/articles/what-properties-do-mixpanels-libraries-store-by-default
+ *
+ * @param {string} data
+ * @return {Object}
+ * @api private
+ */
+
+function formatUserAgent(data){
+  var ret = {};
+  var parsed = parse(data);
+  var browser = parsed.browser;
+  var os = parsed.os;
+
+  if (browser) {
+    ret.$browser = browser.name;
+    ret.$browser_version = browser.version;
+  }
+
+  return ret;
+}
+
+/**
+ * Format mobile specific properties
+ * https://mixpanel.com/help/questions/articles/what-properties-do-mixpanels-libraries-store-by-default
+ *
+ * @param {msg} msg
+ * @return {Object}
+ * @api private
+ */
+
+function formatMobileSpecific(track) {
```

Comment:
Ahh I see, when I meant "Mobile" I was more so saying formatting properties that are mobile related. Not so much referencing the libraries.